.buildkite/nightly.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+  - label: 'Benchmarks'
+    command: |
+        CRITERION_DEBUG=1 cargo bench -- --measurement-time 50 --sample-size 10 nightly
```

Comment:
Tweaking the sample size here is redundant, as it is set directly in the benchmarks.